dependencies-dev

```diff
@@ -3,6 +3,6 @@ Jinja2==3.1.3
 numpy==1.19.5 ; python_version <= '3.9'
 numpy==1.21.6 ; python_version == '3.10'
 numpy==1.23.5 ; python_version == '3.11'
-numpy==1.26.4 ; python_version >= '3.12'
+numpy==2.0.0b1 ; python_version >= '3.12'

```

Comment:
There is fresher version (rc2) already: https://pypi.org/project/numpy/2.0.0rc2/